.code-samples.meilisearch.yaml

```diff
@@ -327,25 +324,25 @@ getting_started_search_md: |-
 
   [About this SDK](https://github.com/meilisearch/meilisearch-js/)
 get_attributes_for_faceting_1: |-
-  client.index('movies').getAttributesForFaceting()
+  client.index('movies').getFilterableAttributes()
 update_attributes_for_faceting_1: |-
   client.index('movies')
-    .updateAttributesForFaceting([
+    .updateFilterableAttributes([
       'genres',
       'director'
     ])
 reset_attributes_for_faceting_1: |-
-  client.index('movies').resetAttributesForFaceting()
+  client.index('movies').resetFilterableAttributes()
 faceted_search_update_settings_1: |-
   client.index('movies')
-    .updateAttributesForFaceting([
+    .updateFilterableAttributes([
       'director',
       'genres'
     ])
 faceted_search_facet_filters_1: |-
   client.index('movies')
     .search('thriller', {
-      facetFilters: [['genres:Horror', 'genres:Mystery'], 'director:Jordan Peele']
+      filter: [['genres:Horror', 'genres:Mystery'], 'director:Jordan Peele']
     })
 faceted_search_facets_distribution_1: |-
```

Comment:
The name of this ID is not accurate anymore, it should be changed depending on the title of the section or the page.